<?xml version="1.0"?>
<launch>

    <!-- robot drivers -->
    <node pkg="jaguar4x4_2014" type="jaguar4x4_2014_node" name="jaguar_drivers" output="screen"/>
    <include file="$(find cameleon_teleop)/launch/cameleon_teleop.launch"/>


    <node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
        <param name="output_frame" value="odom"/>
        <param name="freq" value="30.0"/>
        <param name="sensor_timeout" value="1.0"/>
        <param name="odom_used" value="true"/>
        <param name="imu_used" value="true"/>
        <param name="debug" value="false"/>
        <param name="self_diagnose" value="false"/>

        <remap from="odom" to="raw_odom" />
        <remap from="imu_data" to="drrobot_imu" />
        <!-- <remap from="vo" to="gps_meas" /> -->git
    </node>

    <node pkg="rviz" type="rviz" name="rviz" output="screen"/>

</launch>
